docs/index.rst

```diff
@@ -1,61 +1,138 @@
+   * - **Thursday July 8th**
+     - Leopold Talirz
+     - `The AiiDA Plugin Ecosystem`_
+   * -
+     - Casper Andersen
+     - `AiiDA and OPTIMADE`_
+
+Hands-on sessions
+-----------------
+
+The material for the hands-on sessions is divided in 5 units:
+
+.. toctree::
 
-   Previously organised tutorials
-
-   -------
-   :column: col-lg-6 col-xs-12 pl-0 pr-0
+   sections/getting_started/index
+   sections/running_processes/index
+   sections/managing_data/index
+   sections/writing_workflows/index
+   sections/creating_plugins/index
 
-   .. link-button:: https://aiida-tutorials.readthedocs.io/en/tutorial-2020-intro-week/
-      :type: url
-      :text: In-Depth: 2020 Introductory Virtual tutorial
-      :classes: btn-link btn-block stretched-link
+These are also accessible via the sidebar on the left.
+During the first four days of the tutorial (Monday to Thursday) each of the hands-on sessions will work on the corresponding unit.
 
-   -------
-   :column: col-lg-6 col-xs-12 pl-0 pr-0
+For easy access, you can find the modules for the first hands-on session below:
 
-   .. link-button:: https://aiida-tutorials.readthedocs.io/en/tutorial-qe-short/
-      :type: url
-      :text: Quantum ESPRESSO introductory tutorial
-      :classes: btn-link btn-block stretched-link
+.. panels::
+   :body: bg-light text-center
 
-Videos
-------
+   ------
+   :column: col-lg-6 pl-0 pr-0
 
-For some events we have also recorded the lectures, that you can find here. You will also find the
-links inside the respective sections.
+   .. link-button:: fundamentals-setup
+      :type: ref
+      :text: Getting started: JupyterHub setup
+      :classes: btn-link btn-block stretched-link
 
- *  `2020 AiiDA virtual tutorial (aiida-core 1.3.0) <https://aiida-tutorials.readthedocs.io/en/tutorial-2020-intro-week/#presentations>`_
- *  `2019 tutorial lectures (aiida-core 1.0.0b3) <https://www.youtube.com/playlist?list=PL19kfLn4sO_-QtPaHAA8KByFluT2vvlG0>`_ (`mirror <https://www.bilibili.com/video/av64190843/>`_)
- *  `2019 plugin migration workshop lectures (aiida-core 1.0.0b2) <https://www.youtube.com/playlist?list=PL19kfLn4sO_9laPkLBdv935PquRxX8dCc>`_
+   ------
+   :column: col-lg-6 pl-0 pr-0
 
- .. toctree::
-   :hidden:
+   .. link-button:: fundamentals-basics
+      :type: ref
+      :text: First hands-on: Getting started - AiiDA basics
+      :classes: btn-link btn-block stretched-link
 
-   sections/getting_started/index
-   sections/running_processes/index
-   sections/managing_data/index
-   sections/writing_workflows/index
-   sections/creating_plugins/index
+Contributed talks
+-----------------
+
+Supplementing the presentations above, there are also contributed talks from plugin developers and experienced AiiDA users.
+These can be watched at any time, but please try to watch them **before Friday July 9th**, as we have planned a Q&A with the plugin developers at the start of the first hands-on session of that day.
```

Comment:
Maybe we can bold/highlight those that will be present at the Q&A?

Reply:
Sure, can you remind me again? Emanuele, Togo, Dominik, Leonid (but his talk isn't here yet), ... Anyone else?

Reply:
I'll update this later ;)